tests/Selenium2Config.php

```diff
@@ -16,7 +16,7 @@ public static function getInstance()
      */
     public function createDriver()
     {
-        $browser = $_SERVER['WEB_FIXTURES_BROWSER'];
+        $browser = getenv('WEB_FIXTURES_BROWSER') ?: 'firefox';
```

Comment:
well, using ``getenv`` made it much easier to provide a default value (as I cannot use ``??`` on PHP 5). This is the reason (and we are already using getenv elsewhere in this class anyway).